app/code/Magento/Theme/Model/Theme.php

```diff
@@ -113,6 +119,7 @@ public function __construct(
         \Magento\Framework\View\Design\Theme\ImageFactory $imageFactory,
         \Magento\Framework\View\Design\Theme\Validator $validator,
         \Magento\Framework\View\Design\Theme\CustomizationFactory $customizationFactory,
+        \Magento\Store\Model\App\Emulation $appEmulation,
```

Comment:
Due to [Magento backward-compatible guide](http://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/) we can't add new required parameters to the constructor. For that we can use optional parameters, you can look example in the guide.